src/commands/createFunction.ts

```diff
@@ -102,31 +104,40 @@ export async function createFunction(
     const functionAppPath: string = await workspaceUtil.selectWorkspaceFolder(ui, folderPlaceholder);
     await validateIsFunctionApp(outputChannel, functionAppPath, ui);
 
-    const localAppSettings: LocalAppSettings = new LocalAppSettings(ui, azureAccount, functionAppPath);
-
-    const templatePicks: PickWithData<Template>[] = (await templateData.getTemplates()).map((t: Template) => new PickWithData<Template>(t, t.name));
-    const templatePlaceHolder: string = localize('azFunc.selectFuncTemplate', 'Select a function template');
-    const template: Template = (await ui.showQuickPick<Template>(templatePicks, templatePlaceHolder)).data;
-
-    if (template.bindingType !== 'httpTrigger') {
-        await localAppSettings.validateAzureWebJobsStorage();
+    const languageType: string = workspaceUtil.getProjectType(functionAppPath);
```

Comment:
Why do you infer the project type? I can easily create C# and JavaScript functions in the same project from the command line today.

Shouldn't I be able to create 'Java' and 'JavaScript' functions in the same project from VS Code?

Reply:
How about let the user choose the language he/she wants to add by showing Picks?